autograder/submitty_autograding_shipper.py

```diff
@@ -948,12 +948,11 @@ def checkout_vcs_repo(config, my_file):
                 #    # and check out the right version
                 #    subprocess.call(['git', 'checkout', '-b', 'grade', what_version])
 
-                subprocess.call(['ls', '-lR', checkout_path], stdout=open(checkout_log_file, 'a'))
-                print(
-                    "\n====================================\n",
-                    file=open(checkout_log_file, 'a')
-                )
-                subprocess.call(['du', '-skh', checkout_path], stdout=open(checkout_log_file, 'a'))
+                
+                with open(checkout_log_file, 'a') as log_file:
+                    subprocess.call(['ls', '-lR', checkout_path], stdout=log_file)
+                    print("\n====================================\n",file=log_file)
+                    subprocess.call(['du', '-skh', checkout_path], stdout=log_file)
```

Comment:
These changes are to remove the warnings shown in the Python Autograding Unit Tests part of the CI, and they make the code more consistent, as the rest of the file uses 'with open()' and print().

These change ![Warnings](https://user-images.githubusercontent.com/46759635/219507512-b1af99a0-7130-472e-b26e-1b29de49addb.png)
to 
![updated_error_log](https://user-images.githubusercontent.com/46759635/219507018-120e18b6-da2b-4730-8601-9e68bf8ae8ef.png)